oomph/c_output.py

```diff
@@ -669,7 +712,7 @@ def emit_toplevel_declaration(
                 }}
 
                 // TODO: escaping?
-                struct class_Str *res = cstr_to_string("union {top_declaration.type.name}");
+                struct class_Str *res = {self.emit_string("union " + top_declaration.type.name)};
```

Comment:
The decref below this is no longer correct, because `emit_string` does not return a new reference

Reply:
Actually it is correct, because it does not decref the string coming from `emit_string()`